backend/tests/test_main.py

```
@@ -0,0 +1,32 @@
+"""Stage 1: Basic tests for health endpoints."""
+import pytest
```

Ruff (0.12.2)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)